components/tidb_query/src/expr/builtin_miscellaneous.rs

```diff
@@ -186,6 +187,17 @@ impl ScalarFunc {
             Ok(Some(0))
         }
     }
+
+    pub fn uuid<'a, 'b: 'a>(
+        &'b self,
+        _ctx: &mut EvalContext,
+        _row: &[Datum],
+    ) -> Result<Option<Cow<'a, [u8]>>> {
+        let result = Uuid::new_v4();
+        let mut buf = [b'!'; uuid::adapter::Hyphenated::LENGTH];
```

Comment:
Just another tiny problem.
I think we can simply initial buf using `vec![0; uuid::adapter::Hyphenated::LENGTH]`. Then we don't need to fill the buf at stack and then copy them to the heap.

Reply:
ok, i use vec!